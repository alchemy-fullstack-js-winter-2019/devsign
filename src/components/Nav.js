import React from 'react';
import SearchForm from './SearchForm';
import { StyledLink } from '../styles/mainStyle';
import { NavBar, List, MainLogo, NavLi, NavLink, MobileProfileImage, LogoLink } from '../styles/nav';

function Nav() {
  return (
    <NavBar>
      <List>
        <MainLogo>
          <LogoLink to="/home">Chirp</LogoLink>
        </MainLogo>
        <NavLi>
          <NavLink to="/home">Home</NavLink>
        </NavLi>
        <NavLi>
          <NavLink to="/profile">Profile</NavLink>
        </NavLi>
      </List>

      <StyledLink to="profile">
        <MobileProfileImage alt="profile pic" src="http://capepremierrealty.com/wp-content/uploads/2018/11/generic-profile-icon-7.jpg.png"/>
      </StyledLink>
      
      <SearchForm />
    </NavBar>
  );
}

export default Nav;
