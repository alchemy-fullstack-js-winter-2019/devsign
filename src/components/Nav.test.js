import React from 'react';
import Nav from './Nav';
import { shallow } from 'enzyme';

describe('Nav component', () => {
  it('matches a snapshot', () => {
    const wrapper = shallow(
      <Nav />
    );
    expect(wrapper).toMatchSnapshot();
  });
});
