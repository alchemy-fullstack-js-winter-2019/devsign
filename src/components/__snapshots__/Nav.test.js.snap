// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav component matches a snapshot 1`] = `
<nav
  className="sc-kGXeez dcHRRN"
>
  <ul
    className="sc-kpOJdX eSSkdt"
  >
    <li
      className="sc-gZMcBi sc-ckVGcZ fJmOAI"
    >
      <a
        className="sc-bdVaJa sc-hMqMXs hDuBxd"
        href="/home"
        onClick={[Function]}
      >
        Chirp
      </a>
    </li>
    <li
      className="sc-gZMcBi sc-dxgOiQ btSlKp"
    >
      <a
        className="sc-bdVaJa sc-eNQAEJ gsdyPY"
        href="/home"
        onClick={[Function]}
      >
        Home
      </a>
    </li>
    <li
      className="sc-gZMcBi sc-dxgOiQ btSlKp"
    >
      <a
        className="sc-bdVaJa sc-eNQAEJ gsdyPY"
        href="/profile"
        onClick={[Function]}
      >
        Profile
      </a>
    </li>
  </ul>
  <a
    className="sc-bdVaJa cVFclJ"
    href="/profile"
    onClick={[Function]}
  >
    <img
      alt="profile pic"
      className="sc-jKJlTe hLhmlh"
      src="http://capepremierrealty.com/wp-content/uploads/2018/11/generic-profile-icon-7.jpg.png"
    />
  </a>
  <form
    className="sc-ifAKCX sc-cSHVUG bmJTsc"
  >
    <label
      className="sc-kgoBCf iOxteu"
      htmlFor="search"
    >
      Search
    </label>
    <input
      className="sc-bZQynM sc-kAzzGY kNfhYh"
      id="search"
      readOnly={true}
      type="text"
      value=""
    />
    <label
      className="sc-kgoBCf iOxteu"
      htmlFor="searchButton"
    />
    <button
      className="sc-gzVnrw sc-chPdSV eNmjaL"
      id="searchButton"
    >
      Search
    </button>
  </form>
</nav>
`;
